.jshintrc

```diff
@@ -1,6 +1,4 @@
 {
-  "camelcase": true,
-  "curly": true,
```

Comment:
- We want to keep the 80 character line
- We have since removed the camelcase requirement of our style so we can keep that out
- We want to keep enforcing curly and not doing 1 line if statements